.storybook/preview-head.html

```diff
@@ -1,4 +1,4 @@
 <link
-  href="https://fonts.googleapis.com/css?family=Inter:400;700;900&display=swap"
+  href="https://fonts.googleapis.com/css?family=Inter:400;500;700;900&display=swap"
```

Comment:
we should raise this with the design team. There are too many fonts being downloaded, they should constrain themselves to like 2 (max 3) fonts

Reply:
agreed